src/roving-tabindex/tabindex-provider.js

```diff
@@ -0,0 +1,177 @@
+import React, { createContext, useReducer, useMemo, useEffect } from 'react'
+import PropTypes from 'prop-types'
+
+export const ActionTypes = {
+  REGISTER: 'REGISTER',
+  UNREGISTER: 'UNREGISTER',
+  TAB_TO_PREVIOUS: 'TAB_TO_PREVIOUS',
+  TAB_TO_NEXT: 'TAB_TO_NEXT',
```

Comment:
This is a bit misleading since roving tabindex doesnt use _tabs_ but rather uses arrow keys?

Reply:
that makes sense. I'll update to `MOVE_TO`